### PR TITLE
add licensing to the project

### PR DESCRIPTION
**Pull request by cr1tbit**
Before merging to main, we want to make sure the licensing has no mistake in it.

Pasting the image of the licenses on the PCB for reference:

image

### CODE REVIEW

**Comment by cr1tbit** [comment]
The licensing text:

## Licensing Information
The hardware design files, including schematics and PCB layout (excluding silkscreen artwork and outline design), are licensed under the CERN Open Hardware Licence v2 [CERN-OHL-W](https://ohwr.org/project/cernohl/-/wikis/uploads/f773df342791cc55b35ac4f907c78602/cern_ohl_w_v2.pdf).

Artwork and outline design © 2023 by [Ewa Kielska](https://www.behance.net/dziaberlia55f5) are licensed under [CC BY-NC-ND 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/)

All documentation is released under [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)

In summary, you are free to use schematic and general concept of this project according to `CERN-OHL-W` license. If you want to include the artwork, you must adhere to `CC BY-NC-ND 4.0` too.

We are open to either adjust these licensing terms, or providing the design files with alternative licensing terms in special cases. Feel free to open issue then.
